### PR TITLE
[IMP] make relation selection field

### PR DESCRIPTION
**Pull request by CasVissers-360ERP**
Relation can be a selection field creating cleaner relation information.

### CODE REVIEW

File: hr_employee_related/models/hr_employee_related.py [hunk 1]
```diff
@@ -18,7 +18,18 @@ class HrEmployeeRelated(models.Model):
...
+         ('partner', 'Partner'),
+         ('son', 'Son'),
+         ('daughter', 'Daughter'),
+         ('father', 'Father'),
+         ('mother', 'Mother'),
+         ('brother', 'Brother'),
+         ('sister', 'Sister'),
+         ],
+        string='Employee relation',
+        default='depreciate')
```
Comment (espo-tony) [inline hr_employee_related/models/hr_employee_related.py]: The default value doesn't appear to be in the option list.

**Review by espo-tony — CHANGES_REQUESTED** [review]
Selection field, here, would surely give some advantage in usage, restricting the number of possible options to a small and well definite set.
The drawback is that it makes this field much less flexible, avoiding the possibility to set an unexpected relationship.
I think that, for this case, pros overcome cons, but I'd also like to have a confirm that different relationships than those listed in this PR make no sense in the context of this module.

**Review by espo-tony — APPROVED** [review]

**Comment by pankk** [comment]
Judging from the module's description, this module can be used to store ICE (in case of emergency) information. In that case it would be useful to also have, for example 'caretaker' in the selection. Since there may be other officially designated relations which can serve as ICE contacts, I think it would be prudent to consider adding 'other' to the selection as well.

**Comment by CasVissers-360ERP** [comment]
I agree, added caretaker & other.

**Comment by CasVissers-360ERP** [comment]
an update here?

**Pull request merged**